models/cow_protocol/ethereum/cow_protocol_ethereum_order_rewards.sql

```diff
@@ -1,16 +1,18 @@
 {{ config(alias='order_rewards',
+        tags = ['dunesql'],
         post_hook='{{ expose_spells(\'["ethereum"]\',
                                     "project",
                                     "cow_protocol",
                                     \'["bh2smith"]\') }}'
 )}}
 
 -- PoC Query here - https://dune.com/queries/1752782
-select
-    distinct order_uid,
-    block_number,
-    tx_hash,
-    solver,
-    data.amount as cow_reward,
-    cast(data.surplus_fee as double) as surplus_fee
-from {{ source('cowswap', 'raw_order_rewards') }}
+SELECT
+  DISTINCT from_hex(order_uid) as order_uid,
+  block_number,
+  from_hex(tx_hash) as tx_hash,
+  from_hex(solver) as solver,
+  from_hex(data.quote_solver) as quote_solver,
+  cast(data.amount as uint256)  AS cow_reward,
```

Comment:
Do we have the cow_reward and surplus_fee for backwards compatibility reasons here?

Reply:
`surplus_fee` still exists as far as I understand and yea I guess you cold say `cow_reward` is for backwards compatibility... Also note that the back end is still populating this field... with zeros I think.